src/components/mol.header/header.jsx

```diff
@@ -1,33 +1,40 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { AppIcon } from '../atm.app-icon/app-icon'
 import { Icon } from '../atm.icon/icon'
 import { HBox } from '../mol.box/box.styled'
 import { IconSize } from '../constants'
 import { HeaderStyled } from './header.styled'
 import { H1 } from '../atm.typography/typography.styled'
+import { Menu } from '../mol.menu/menu'
 
 export const Header = (props) => {
+    const [toggleMenu, setToggleMenu] = useState(false);
+    const openMenu = () => {
+        setToggleMenu(!toggleMenu)
+    }
     return (
-        <HeaderStyled height={'medium'}>
-            <HBox grow={true}>
-                <HBox grow={true} hAlign={'flexStart'}>
-                    <HBox.Item onClick={props.handleMenuClick}>
-                        <Icon.Menu size={IconSize.Large} />
-                    </HBox.Item>
-                </HBox>
-                <HBox.Item halign={'center'}>
-                    <AppIcon.Logo />
-                </HBox.Item>
-                <HBox grow={true} hAlign={'flexEnd'}>
-                    <HBox.Item>
-                        <Icon.Cart size={IconSize.Large} />
-                        <HBox.Separator />
-                        <Icon.Cart size={IconSize.Large} />
+        <>
+            <HeaderStyled height={'medium'}>
+                <HBox grow={true}>
+                    <HBox grow={true} hAlign={'flexStart'}>
+                        <HBox.Item onClick={openMenu}>
+                            <Icon.Menu size={IconSize.Large} />
+                        </HBox.Item>
+                    </HBox>
+                    <HBox.Item halign={'center'}>
```

Comment:
halign -> hAlign acho (to achando q vai querer typescript ja ja hahah)
mas como n tem grow, acho q n precisamos desse hAlign certo? (n teria pra onde centralizar)

Reply:
Nossa muito verdade! hahahaha não percebi que escrevi errado e ele ainda não tinha necessidade, porque ele vai ocupar o espaço que só serve pra ele pois não tem grow.